libraries/ui-library/src/components/six-input/six-input.tsx

```diff
@@ -228,6 +228,18 @@ export class SixInput {
     return this.nativeInput?.setSelectionRange(selectionStart, selectionEnd, selectionDirection);
   }
 
+  /** Returns the start and end positions of the text selection */
+  @Method()
+  async getSelectionRange(): Promise<{
+    selectionStart: number | undefined | null;
```

Comment:
Is the differentation between `undefined` and `null` needed? If not, I suggest to change it to `number | null;` (same as in the native html)

Reply:
Yes and no. The native input returns `number | undefined | null` as possible types. I have changed the return type and if undefined, will return null